dashboard/views.py

```diff
@@ -1604,7 +1604,7 @@ def get_form(self):
         form = super().get_form()
 
         # Decide language options
-        available_languages = self.answer.list_available_languages()
```

Comment:
Why is this line showing up here? It's not relevant for this PR, is it?

Reply:
no its relevant to this PR as these changes are checked so, i put it up here so the testing of the PR can be done easily also for article translation too.